Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.0.0'
```

Comment:
http://docs.travis-ci.com/user/languages/ruby/

It seems we can set the Ruby version on `.travis.yml` or `.ruby-version`. I think that is a better alternative, no?